src/main/java/com/epam/bookstoreservice/mapper/BookResponseDTOAssembler.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.bookstoreservice.mapper;
+
+import com.epam.bookstoreservice.controller.BookStoreController;
+import com.epam.bookstoreservice.dto.response.BookResponseDTO;
+import com.epam.bookstoreservice.model.BookModel;
+import org.springframework.hateoas.server.RepresentationModelAssembler;
+import org.springframework.stereotype.Component;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+/**
+ * model assembler for BookResponseDTO
+ */
+@Component
+public class BookResponseDTOAssembler implements RepresentationModelAssembler<BookResponseDTO, BookModel> {
```

Comment:
You could move this assembler to a new package called 'hateoas/assembler' since this feature is related to it.